libsolidity/interface/CompilerStack.h

```diff
@@ -454,6 +454,7 @@ class CompilerStack: boost::noncopyable
 	langutil::EVMVersion m_evmVersion;
 	smtutil::SMTSolverChoice m_enabledSMTSolvers;
 	std::map<std::string, std::set<std::string>> m_requestedContractNames;
+	std::map<ContractDefinition const*, std::string const> m_otherYulSources;
```

Comment:
I did say "member" first, but tried to correct myself to "or passed down" :-).
I think it would be better to only keep it during ``generateIR``, just like the second argument for ``compileContract``.
Otherwise, we'd probably need to clear this in ``reset()`` and think about whether it's valid to keep it otherwise - I think passing it down as an argument it less of a headache.